docs/create-packages/Sign-a-Package.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to Create a Localized NuGet Package | Microsoft Docs
+author: rido-min
+ms.author: rido-min
+manager: unniravindranathan
+ms.date: 02/23/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: Explains how signed packages can be used to enable content integrity verification
+keywords: NuGet package signing, NuGet security, creating signed packages
+ms.reviewer:
+- karann-msft
+- anangaur
+---
+
+# Signed packages
+
+NuGet packages can include a digital signature which provides protection against tampered content. This signature is produced from an x509 certificate that also adds authenticity proofs to the actual origin of the package. 
+
+> [!Warning] 
+> NuGet.org does not accept signed packages yet. This feature will be announced using the [announcements](https://github.com/NuGet/Announcements) github repository.
+
+## Types of signed packages
+
+NuGet supports two types of signatures:
+
+- **Author Signatures** are created by the package author before the package is being submitted to any repository. These signatures provide validation end to end, from authors to consumers independent from which repository the package has been obtained.
+- **Repository Signatures** provide an integrity guarantee for all packages in a repository whether they are author signed or not, even if those packages are obtained from a different location than the original repository where they were signed.
+
+Considering author and repository signatures there are four possible package signing configurations; no other configurations of these signature types are allowed:
+
+|Package Type|Notes    |
+|------------|---------|
+|Unsigned|The package does not include any signature.|
+|Author|The primary signature is an author signature.|
+|Repo|The primary signature is a repository signature.|
+|Author and Repo|The primary signature is an author signature, and the signature is repository countersigned.|
+
+> [!Note] 
+> Package repositories can host packages of any of these four types, however once NuGet.org implements repository signatures, all packages in NuGet.org will fall in the last two types.
+
+The technical specification for NuGet package signatures can be found in https://github.com/NuGet/Home/wiki/Package-Signatures-Technical-Details. 
+
+### Benefits of author-signed packages
+
+Packages that have been signed with an author certificate provide the strongest end to end validation, since the author signatures guarnatees that the package has not been modified since it leaves the author machine, until it is downloaded into the consumer machine, no matter fro which repository or transport method has been used.
```

Comment:
Edit:

Packages that have been signed with an author certificate provide the strongest end to end validation. An author signature guarantees that the package has not been modified since it left the author computer, no matter fro which repository or transport method has been used."

It doesn't seem like the downloading step is relevant here?